build_for_ultimaker.sh

```diff
@@ -43,13 +43,138 @@ DEB_DIR=`pwd`/debian
 KCONFIG=`pwd`/configs/${BUILDCONFIG}_config
 KERNEL_BUILD=`pwd`/_build_armhf/${BUILDCONFIG}-linux
 
+INITRAMFS_MODULES="sunxi_wdt.ko"
+INITRAMFS_COMPRESSION="${INITRAMFS_COMPRESSION:-.lzo}"
+INITRAMFS_ROOT_GID=${INITRAMFS_ROOT_GID:-0}
+INITRAMFS_ROOT_UID=${INITRAMFS_ROOT_UID:-0}
+INITRAMFS_SOURCE="${INITRAMFS_SOURCE:-initramfs/initramfs.lst}"
+INITRAMFS_IMG="${KERNEL_BUILD}/initramfs.cpio${INITRAMFS_COMPRESSION}"
+
+GEN_INIT_CPIO="${KERNEL_BUILD}/usr/gen_init_cpio"
+GEN_INITRAMFS_LIST="${KERNEL}/scripts/gen_initramfs_list.sh"
+
+BB_PKG="http://ftp.nl.debian.org/debian/pool/main/b/busybox/busybox-static_1.22.0-19+b3_armhf.deb"
+BB_BIN="busybox"
+
+DEPMOD="${DEPMOD:-/sbin/depmod}"
+if [ ! -x "${DEPMOD}" ]; then
+    DEPMOD="busybox depmod"
+    if [ ! -x "${DEPMOD}" ]; then
+        echo "No depmod binary available. Cannot continue."
+        exit 1
+    fi
+fi
+
 # Set the release version if it's not passed to the script
 RELEASE_VERSION=${RELEASE_VERSION:-9999.99.99}
 
 # Initialize repositories
 git submodule init
 git submodule update
 
+busybox_get()
+{
+    local BB_DIR=$(mktemp -d)
+    local BB_AR="${BB_DIR}/busybox-static_armhf.deb"
+    local INITRAMFS_DIR="${1}"
+    local cwd=$(pwd)
```

Comment:
Maybe use pushd and popd for directory switching

Reply:
pushd and popd are bashism, which are not available in ash (or are not part of the posix shell)
On a personal note, we should avoid bashisms as much as possible so that changing the shell is never a problem ;)